docs/redis.md

```diff
@@ -80,11 +80,13 @@ which is based on the [`--fork` upgrade method](https://devcenter.heroku.com/art
     This starts at `fork in progress` and is supposed to change once completed
     (forks start as replicas and then switch to primaries), but it may appear
     stuck in that state. If that happens, it should be safe to continue as long
-    as all data looks to be transferred. Do this by inspection of:
+    as all data looks to be transferred. Do this by entering Redis CLI (`heroku
+    redis:cli`) on both instances and comparing the output of:
```

Comment:
`heroku redis:cli` used to not connect over TLS, which was bonkers, so for a long time I've used [my own little program](https://gist.github.com/tsibley/6f82bcaadf25869b0251f68d8be8d9b5) wrapping the Heroku CLI and standard Redis CLI.

But I just checked again now, and at least as of `heroku version`

    heroku/7.60.2 linux-x64 node-v14.19.0

it does indeed connect over TLS.

I think if you want to use more advanced features of the Redis CLI, you'll still need to invoke it yourself directly (`heroku redis:cli --help` doesn't indicate it will pass thru any args), but it's nice that for a simple REPL you can just use `heroku redis:cli` now.